tap_slack/client.py

```diff
@@ -30,6 +31,50 @@ def get_next(self, response: Response) -> str | None:
         return super().get_next(response)
 
 
+class ThrottledPageNumberPaginator(BasePageNumberPaginator):
+    """Paginator for Slack Logs."""
+
+    max_requests_per_minute = 200
+
+    def has_more(self, response: Response) -> bool:
+        """Has more pages.
+
+        Args:
+            response: API response object.
+
+        Returns:
+            Boolean flag used to indicate if the endpoint has more pages.
+        """
+        data = response.json()
+        max_page = data["paging"]["pages"]
+        current_page = data["paging"]["page"]
+        has_more = True
+        if current_page == max_page:
+            has_more = False
+        return has_more
```

Comment:
```suggestion
        return current_page != max_page
```

though this implementation of `has_more` is redundant with `get_next` and probably unnecessary. I could be wrong, though.